### DISCUSSION TITLE
Feature Request: Session Recording

### DISCUSSION THREAD

**Discussion by chinthakadeshapriya** [edumeet/edumeet#855] [Ideas]
It is a valuable feature if we have session recorded to the cloud storage

**Comment by bpcurse** [comment]
Recording is a feature that has to be well planned and implemented regarding all the possible privacy implications and legal issues.
Don't make the same mistake(s) as BigBlueButton did: https://github.com/bigbluebutton/bigbluebutton/issues/9202
https://github.com/bigbluebutton/greenlight/issues/1163

**Comment by misi** [comment]
See #212  maybe it is duplicate

**Comment by abakanindex** [comment]
Hi there, I need this feature, please suggest how to implement recording on server side. I found [this](https://github.com/muaz-khan/RecordRTC/tree/master/RecordRTC-to-Nodejs) which is working locally. How integrate this with MM? Any help or suggestion most welcome

Regard & thanks
